modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/EditAppToolbar.es.js

```diff
@@ -174,19 +186,9 @@ export default function EditAppToolbar({isSaving, onCancel, onSave}) {
 
 				<ClayButton.Group className="ml-2">
 					<ClayButton
-						disabled={!app.active && isDisabledSaveButton}
+						disabled={!app.active && disabledSaveButton}
 						displayType={app.active ? 'secondary' : 'primary'}
-						onClick={
-							app.active
-								? onClickUndeploy
-								: config.formView.missingRequiredFields
-										?.customField && appId
-								? () =>
-										setMissingRequiredFieldsModalProps({
-											visible: true,
-										})
-								: () => setDeployModalVisible(true)
-						}
+						onClick={getManageAppStatusOnClick}
```

Comment:
@jotascript you need to pass the execution of the function, in this case. Like that: `getManageAppStatusOnClick()`